### PR TITLE
Update Beam's recommended broadcast settings

### PR DESCRIPTION
**Pull request by ttaylorr**
These changes reflect the broadcast settings that we currently recommend.

### CODE REVIEW

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

**Comment by Gol-D-Ace** [comment]
Please prefix your commit with rtmp-services:
For example rtmp-services: Update Beam's recommended settings
Thank you. @ttaylorr

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -295,7 +295,10 @@
             }
         ],
         "recommended": {
-            "max video bitrate": 3500
+            "keyint": 1,
+            "max audio bitrate": 128,
+            "max video bitrate": 3500,
```
Comment (Gol-D-Ace) [inline plugins/rtmp-services/data/services.json]: The , here is invalid.

**Comment by ttaylorr** [comment]
Thank you for letting me know, I reworded my commit message to reflect this.

**Comment by jp9000** [comment]
Change the max audio bitrate to 160

**Comment by ttaylorr** [comment]
@jp9000: Sure, the max audio bitrate is now set to 160.

**Comment by jp9000** [comment]
wait a second, keyint 1?  that doesn't seem right at all.  That's terrible.  That can't be right.

**Comment by ttaylorr** [comment]
`keyint` now at 2.

**Comment by jp9000** [comment]
Alright, thank you.  Where are you getting this information from by the way?  I feel like this should have CBR on as well.  The settings feel a slight bit inconsistent with normal streaming settings.   Are they using HLS for server -> viewer?  That's typically why the low keyframe interval is required, but 1 second would definitely be weird

**Comment by ttaylorr** [comment]
I am a developer on the Beam project (can be verified by my presence in the GitHub organIzation: @WatchBeam).  Our ingest servers use RTMP, but we actually communicate from our distribution servers to client by sending the mp4 stream over a secure websocket connection, and a keyframe interval of 1 second works best there.  To answer your question about CBR, we don't currently enforce that users stream at a constant bitrate, which is why I've omitted it from the recommended settings.

**Comment by jp9000** [comment]
Ah, my apologies for the misunderstanding then, and if I seemed rather forward, I should have looked at your profile from the beginning.

160 audio bitrate should be fine, right?  We use that for default, and the reason is because we defer to FFmpeg's experimental AAC encoder if none other are available, which has pretty terrible quality if you go too low, and relative to the entire stream it's only a tiny bit more bitrate to possibly reduce audio artifacting in that case, so I feel like 160 feels just about right to compensate (though honestly I probably couldn't tell).  If however for technical reasons you need 128 then it would be more understandable, but it seems like you're okay with it so I appreciate it.

The keyframe interval thing was due to the fact that we're used to services using HLS, which usually operates at 2-second keyframe intervals, hence my confusion.  However, even with the fact that you're doing a different method, I can't help but feel it will start negatively affecting video quality if you go too low with the keyframe interval.  When Twitch and Youtube initially started these requirements there was a big uproar about it due to the fact that it couldn't really compress as well, though from a networking standpoint it's fairly understandable due to the need to cache smaller usable chunks.  Halving the keyframe interval once again to 1 from 2 feels like it'll lose more than it ultimately gains.  Is 2 seconds going to be okay for what you're doing?  I didn't mean to encroach on your recommendations, I was basing it upon what other streaming services were doing.  If you feel it's necessary to change it back I understand, but for quality reasons I would definitely prefer it if it didn't have to be at 1.

Also, I feel like CBR probably should be on for a few reasons, but primarily for the fact that the user might be forced to mess with their CRF value to adjust the quality relative to their bitrate manually if it's off.

**Comment by jp9000** [comment]
You still there @ttaylor?  Is 160 audio bitrate and 2-second keyframe interval alright for you?  If so I'll merge.  Again I didn't mean to step on your recommendations, I didn't know who you were at the time, but we'd definitely prefer to have those settings.

**Comment by ttaylorr** [comment]
Sorry, still talking with the rest of the team to see what's what :smile:

**Comment by Doridian** [comment]
Greetings, another of the @WatchBeam team here.
First of all, thanks for your feedback about our suggested values/settings.

We had some discussions among our team about these settings and we have come to the following conclusion: The 160 bitrate along with a 2 second keyframe interval is alright and will work fine with our technology stack.

CBR should also be recommended to be on as you suggested to ensure an easier experience for our users.

So feel free to go ahead and merge this in.

**Comment by ttaylorr** [comment]
Thanks for weighing in here, @Doridian.

I've gone ahead and enabled CBR for Beam in the services file, which makes this PR ready to merge.  Thanks again for all of your help and recommendations here, @jp9000! :smile:

**Comment by jp9000** [comment]
Alrighty, thank you.

**Pull request merged**